app/views/branded/public_pages/_plans_facet_form.html.erb

```diff
@@ -15,7 +15,11 @@
       <button aria-label="search" type="submit"><i class="fas fa-search"></i></button>
     </div>
     <% search_msg = search.present? ? _('Showing results for "%<search_term>s"') % { search_term: sanitize(search) } : '' %>
-    <div id= "js-search-results-block" class="js-search-results c-search-results"><%= search_msg %></div>
+    <div id= "js-search-results-block" class="js-search-results c-search-results">
```

Comment:
@briri - What's interesting is that there should really be a label with that Search input field. We should probably use "aria-labelledby=search-label" in the input field as well, so that the user knows that that input field is a Search field.

Reply:
it actually does. There is an `aria-labelledby` that points to the `<h3 id="search-label"><%= _('Search') %></h3>` above it

Reply:
@briri - oh nice. I totally missed that. Thanks.